### ISSUE TITLE
Allow users to scale an existing Deployment

### ISSUE THREAD

**Issue by arschles** [kedacore/http-add-on#35]
### Use-Case

Currently you can't start scaling an existing `Deployment`, you can only create a new one by submitting a new `HTTPScaledObject`. It would be helpful to allow someone to submit a new `HTTPScaledObject` to begin scaling an already existing `Deployment`.

This idea came from #31 in [this comment](https://github.com/kedacore/http-add-on/pull/31/files#r568569492)

### Specification

- [ ] User should be able to specify an existing `Deployment` in an `HTTPScaledObject`
    - They will also need to specify the `Service` name and port to which traffic should be forwarded to their app
- [ ] KEDA and the HTTP add on should begin routing traffic to their app and scaling based on HTTP traffic

**Comment by tomkerkhove** [comment]
Any progress on this? I'd say this should be one of our priorities as majority of users would prefer to use this approach (certainly for existing workloads).

**Comment by khaosdoctor** [comment]
I just started #36, I can change the API to add this too, but I'm out of ideas on how to do it the best way.

**Comment by arschles** [comment]
@khaosdoctor I'm good with that. let's split this work out into a separate PR from #36 though. let me know if you'd like me to do one of these

**Comment by khaosdoctor** [comment]
I think this one would be after #36 since it'd need the new API to work. If you want to do it I'm fine with that :)

**Comment by khaosdoctor** [comment]
Actually, since this is on the alpha2 milestone I'll get this going ASAP so the API is ready for #36
